recipes/libpfm4/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conan import ConanFile, tools
+import os
+
+
+class Libpfm4Conan(ConanFile):
+    name = "libpfm4"
+    license = "MIT"
+    homepage = "https://github.com/wcohen/libpfm4"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = (
+        "a helper library to program the performance monitoring events")
+    topics = ("perf", "pmu", "benchmark", "microbenchmark")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    # Sources are located in the same place as this recipe, copy them to the recipe
+    exports_sources = "src/*"
+    _source_subfolder = 'sources'  # Required to build at least 2.12.1
+
+    def source(self):
+        from pprint import pprint
+        pprint(self.conan_data)
+        tools.files.get(self,
+                        **self.conan_data['sources'][self.version],
+                        strip_root=True,
+                        destination=self._source_subfolder)
+
+    def config_options(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def build(self):
+        with tools.files.chdir(self, self._source_subfolder):
+            self.run('make')
+
+    def package(self):
+        make_params = {
```

Comment:
Use AutotoolsToolchain instead.

Reply:
Note that there are no autotools involved here. It's a pure `make` project. Would `AutotoolsToolchain` still work?

Reply:
Yes.

Reply:
Took your PR. Thanks.